### ISSUE TITLE
var url = 'http://localhost:3099/';

### ISSUE THREAD

**Issue by LaxmanFareportal** [braintree/popup-bridge-ios#11]
### General information

Could you please explain what do you mean by  step 

4. Use PopupBridge from the web page by writing some JavaScript:

5. Redirect back to the app inside of the popup:

How to do these steps on iOS app? 

Explain 

* SDK/Library version: 
* iOS Version and Device: 
* Integration type and version:

**Comment by crookedneighbor** [comment]
Are you trying to use Popup Bridge with PayPal, or just generally with popups? The instructions you're referencing are for general use.

For reference, the [documentation for integrating with PayPal is here](https://github.com/braintree/popup-bridge-ios#using-paypal-in-a-webview). Most of the work to integrate with PayPal is already done in the Braintree and PayPal SDKs.

For general usage, the code snippets below the steps you reference illustrate what to do.

> 4. Use PopupBridge from the web page by writing some JavaScript:
>
>```js
> var url = 'http://localhost:3099/';
>
>  if (window.popupBridge) {
>  // Open the popup in a browser, and give it the deep link back to the app
>  popupBridge.open(url + '?popupBridgeReturnUrlPrefix=' + popupBridge.getReturnUrlPrefix());
>```

Where `url` is the popup you are opening. And you append `'popupBridgeReturnUrlPrefix=' + popupBridge.getReturnUrlPrefix()` as a query string to that url so popup bridge knows how to return your user back to your webview.

> 5. Redirect back to the app inside of the popup:
>```js
>if (location.search.indexOf('popupBridgeReturnUrlPrefix') !== -1) {
>    var prefix = location.search.split('popupBridgeReturnUrlPrefix=')[1];
>    // Open the deep link back to the app, and send some data
>    location.href = prefix + '?color=' + color;
>```

Here it checks if that query parameter exists, and if it does it changes the location of the popup back to the webview.

Does that clear it up for you?

**Comment by LaxmanFareportal** [comment]
I am trying to use Popup Bridge with PayPal. Have downloaded the Example app and tried to run this code. It shows blank on Simulator. 

what is this URL for http://localhost:3099/ ?

We are planning to implement Paypal using webview. Whats the best document to follow? Please advise. 

REQ:

We need URL that opens paypal screen and takes the control of payment. Once the payment is done it returns the control back to app. How can this be achieved ?

**Comment by crookedneighbor** [comment]
All you need to do is follow steps 1-3 and create a webpage opened in a WKWebview that uses [PayPal's Checkout.js](https://github.com/paypal/paypal-checkout). Everything else is handled by the Checkout.js SDK.

**Comment by LaxmanFareportal** [comment]
Have completed 1-3 steps. 

Please explain what this URL actually is? I am getting blank page on my app. 
http://localhost:3099/

Also if we run the popupbridgeexample target. The simulator shows blank and nothing really happens. Could you please fix that issue. May be then we can compare with the example code. 

Thanks,
Laxman

**Comment by LaxmanFareportal** [comment]
What should be this URL in my app? 
http://localhost:3099/

**Comment by crookedneighbor** [comment]
The Popup Bridge library can be used generically with any popup, not just PayPal. `http://localhost:3099` is supposed to represent any popup you are trying to access with Popup Bridge. We will update the README documentation to make that clearer.

Here's a gif of what should happen when you open the demo:

![popupdemo](https://user-images.githubusercontent.com/2916945/40373039-a5f79dd6-5dab-11e8-8df0-e46e2700e7c8.gif)

Not sure why it doesn't work for you.

If you're a Braintree merchant and need further assistance with your integration, please contact our support team https://articles.braintreepayments.com/forms/contact

If you're not a Braintree merchant, I recommend reaching out to the support channels through PayPal.

**Comment by crookedneighbor** [comment]
Correction, `http://localhost:3099/` refers to the webpage you are opening in the webview. Not the popup. So in this case, you'd substitue `http://localhost:3099/` for your website's checkout page that has checkout.js on it.
